addons/towing/initSettings.sqf

```diff
@@ -0,0 +1,17 @@
+[
+    QGVAR(addRopeToVehicleInventory), "CHECKBOX",
+    LSTRING(Setting_addRopeToVehicleInventory_DisplayName),
+    LELSTRING(OptionsMenu,CategoryLogistics),
+    true,
+    true,
+    {
+        if (!_this) exitWith {
+            [QGVAR(addRopeToVehicleInventory), _this] call EFUNC(common,cbaSettings_settingChanged);
+        };
+        if (!isServer || {!isNil QGVAR(addRopeToVehicleInventory_initialized)}) exitWith {};
+        GVAR(addRopeToVehicleInventory_initialized) = true;
+        ["Tank", "initPost", {call FUNC(addRopeToVehicle)}, true, [], true] call CBA_fnc_addClassEventHandler;
+        ["Car", "initPost", {call FUNC(addRopeToVehicle)}, true, [], true] call CBA_fnc_addClassEventHandler;
```

Comment:
For CBA XEH, don't need the `call` wrapper optimization like BI EHs. Can just use `FUNC` directly or `LINKFUNC` (preferred).

Reply:
forgot about `LINKFUNC`. Will fix in 2 weeks (can't test atm) if you don't fix earlier